### DISCUSSION TITLE
Update Bromite / Update not available

### DISCUSSION THREAD

**Discussion by Hypofant** [bromite/bromite#2163] [General]
I'm confused, being a absolute beginner.
My bromite version is 102.0.5005.96
Allow check for updates is enabled.
But it says: Update Bromite. Update not available.

Your current version might be vulnerable.

What can I do? Just wait? Load another version (which?)

**Comment by csagan5** [comment]
There is a link in the popup, you can click it and read the explanation: https://www.bromite.org/vulnerable-version

**Reply by rusty-snake** [reply]
> you should stop using it until there is a new update

Is this the official recommendation? This renders bromite useless if you have to stop using it every month for two weeks. Or is this long waiting for 103 an exception and updates are usually faster?

I understand that it is a fork and being maintained by a few people (in their free time?) who do huge and great work. However waiting two weeks for security patches in a critical system component like the browser is not justifiable from a security point of view.

**Reply by SkewedZeppelin** [reply]
https://divestos.org/misc/ch-dates.txt

**Reply by csagan5** [reply]
> Is this the official recommendation? This renders bromite useless if you have to stop using it every month for two weeks. Or is this long waiting for 103 an exception and updates are usually faster?
> 
Yes, it is the official recommendation. You can figure out by yourself it is secure or not (by checking if the CVEs are applicable) or stop using it until an update is available.

> I understand that it is a fork and being maintained by a few people (in their free time?) who do huge and great work. However waiting two weeks for security patches in a critical system component like the browser is not justifiable from a security point of view.

This is the best I can do and the alternative is to close down the project; not planning to do that any time soon but at least for awareness and transparency there is a notification for the users.

**Reply by rusty-snake** [reply]
> You can figure out by yourself it is secure or not (by checking if the CVEs are applicable) 

G usually gives not much details about CVEs in chrome to the public¹ which makes this check very hard. 

¹ initially, of course they publish more details but only after some time.

**Reply by NineOutOfTen** [reply]
Dear CSagan5,

It is unfortunate that this has happened. I like your browser and I hope you are able to update Bromite soon. In my view, you and the others involved in this effort should get paid rather than volunteer. 

I suggest you consider conducting a crowdfund campaign and get paid for doing this. Creating and maintaining a browser is a lot of work and what you're going through now, in many ways, doesn't surprise me.

Please consider my suggestion. At least by getting paid it will make it easier for you and others involved. Good luck!

**Reply by csagan5** [reply]
@NineOutOfTen you can always help the project or fork it and make it better, that is why I chose to make it open source. You do not have to wait on others to do anything for you.

**Reply by rusty-snake** [reply]
@vortaj Writing a two sentence response is done in a minute from a mobile device. Rebasing, Testing and Updating patches requires a lot of time and work, especially for monster code-base like chromium has.

**Reply by NineOutOfTen** [reply]
> @NineOutOfTen you can always help the project or fork it and make it better, that is why I chose to make it open source. You do not have to wait on others to do anything for you.

I have neither the knowledge nor the skills to do it. Are you opposed to crowdfunding your project? In my view, your not doing so or having funding sources of some kind was a huge mistake.

Browsers like Firefox and Brave are successful because they are backed by organizations with lots of money and people who can maintain them and create new features.

**Reply by csagan5** [reply]
> I have neither the knowledge nor the skills to do it.

Then you will have to wait like all Bromite users, me included.

> Are you opposed to crowdfunding your project? In my view, your not doing so or having funding sources of some kind was a huge mistake.

Thanks for your opinion. This project is not crowd-funded; there are already different ways to help this project however money is not the most important and I wish to keep it that way.
